packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -269,6 +274,11 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         })}
         id={uniqueId}
         aria-labelledby={`${props.ariaLabelledBy} ${uniqueId}`}
+        aria-label={
+          menuItems && menuItems.length > 0
+            ? ariaDisplayName
+            : strings.disabled.replace('{user}', ariaDisplayName || '')
```

Comment:
Is using [aria-disabled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled) not sufficient?

Reply:
did a little bit of digging and yes I htink aria-disabled=true would work here as well!
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

@JoshuaLai , my bad! If this works, then this is probably the better approach here

Reply:
Fixed up thanks for the call out. Using aria-disabled on the menu-item div works. Also as a note on Windows voice over says "Disabled" on MacOS voice over says "Dimmed"